index.js

```diff
@@ -49,6 +51,11 @@ parse.y = parse.d * 365.25
  */
 
 function parse(str){
+  
+  if(!/\d/g.test(str)){
```

Comment:
Maybe this makes sense only for strings without `s` at the end? Because it will be illogical to get `1000` for `seconds`, but for `second` it's ok